.travis.yml

```diff
@@ -23,3 +23,4 @@ env:
   - secure: R3N94P7noue7/K2KOLjfxNVZV6Eft2emrU0zfh/qbkAwW6CF6w3gjVxRJwBzOhrsy5W4SU7r18Dft9/YS1QQOKw5HuBGNKnVytJxhkHPRB7EE/KrDnlQ0uHK9LAlQnwQzb74+iyQv6ONcrCF1M7xEnnkst8buZp4zoB8JXUA8bM=
   - secure: jh4RLrx+HoTmzgfvsxle8hWdAsz6m5M0qvaBu3VAGPnrM4NmRPyfyb08btlN+qLJOTEBilZGIKh+cYSE8NRJHXR+8iAKHikop8rOh6HqwaEl8A7SFy/Nx6RMys3Cy3yX5AVLHhgpu4absok4QlVplYhn2aK6EbYsM0VS1uyOztQ=
   - secure: ZdJH16Iz79TTwPCha92BuekZP07QsLAxMqaG4liH45ncK4TE+xfEE0lAmt2SqonPodXDbx94cFwjTEum0xFsEOp1ASsz393i265+qVKT+WwumpMe1/4Ta9UF6bAZMBQSeikgW7Qnh7uN2DVUKJ9LNrCg3xlBReWeqJC/1AjD+TM=
+  - ENCRYPTION_LABEL: <placeholder>
```

Comment:
Shouldn't this be removed if we're not using it?

On the other hand, should we be using it, to make the GH publishing consistent across projects?

Reply:
Yes! You're right